spec/lib/direct_message_channel_creator_spec.rb

```diff
@@ -64,15 +64,15 @@
       expect(user_2_membership.desktop_notification_level).to eq("never")
```

Comment:
Yep, working in this really did highlight the limits of too much acceptance. Here we are testing a lib, and testing it as a high level acceptance, we should be testing public methods and their various cases, otherwise you end up with this kind of things: `it "tests this and that, and also this when these are true"`